docs/running-without-go.md

```diff
@@ -3,15 +3,18 @@
 `gotestsum` may be run without Go as long as the package to be tested has
 already been compiled using `go test -c`, and the `test2json` tool is available.
 
-The `test2json` tool can be compiled from the Go source tree:
+The `test2json` tool can be compiled without installing go:
 
 ```sh
-export GOVERSION=1.13.5
-curl -Lo go.zip "https://github.com/golang/go/archive/go${GOVERSION}.zip"
-unzip go.zip
-rm -f go.zip
-cd go-go${GOVERSION}/src/cmd/test2json/
-env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w" .
+GOVERSION=1.13.5
+OS=$(uname -s | sed 's/.*/\L&/')
+mkdir -p gopath
+GOPATH=$(realpath gopath)
+HOME=$(realpath ./)
+curl -L --silent https://go.dev/dl/go${GOVERSION}.${OS}-amd64.tar.gz | tar xz -C ./
+env HOME=$HOME GOOS=linux GOARCH=amd64 CGO_ENABLED=0 GOPATH=$GOPATH ./go/bin/go build -o test2json -ldflags="-s -w" cmd/test2json
+rm -rf go
+rm -rf gopath
```

Comment:
```suggestion
```

Let's leave any cleanup out of these instructions so prevent anyone from accidentally deleting something they wanted to keep.